internal/consensus/replay_test.go

```diff
@@ -297,7 +297,7 @@ type simulatorTestSuite struct {
 	GenesisState sm.State
 	Config       *config.Config
 	Chain        []*types.Block
-	Commits      []*types.Commit
+	ExtCommits   []*types.ExtendedCommit
```

Comment:
Not sure we really needed to change the field name—they're still commits, they're just more exuberant than before. 🙂